src/mobile/src/components/Post/Posts.jsx

```diff
@@ -0,0 +1,53 @@
+import Constants from 'expo-constants';
+import { FlatList, View, Button, StyleSheet } from 'react-native';
+import useSWRInfinite from 'swr/infinite';
+import Post from './Post';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  divider: {
+    backgroundColor: '#9f9f9f',
+    height: 1,
+    marginHorizontal: 'auto',
+    marginVertical: '1rem',
+    width: '80%',
+  },
+});
+
+const POST_URL = 'https://dev.api.telescope.cdot.systems/v1/posts'; //Constants.manifest.extra.postsURL;
+
+function Posts() {
+  const {
+    data: pages,
+    error,
+    setSize,
+    size,
+  } = useSWRInfinite((index) => `${POST_URL}?page=${index + 1}&per_page=2`);
```

Comment:
```suggestion
  } = useSWRInfinite((index) => `${POST_URL}?page=${index + 1}`);
```

Reply:
will change right after we got this PR aprroved

Reply:
Why don't we change in this PR @tpmai22?

Reply:
I mean I want to make ease for ppls who trying to review this after I got all the approve. I will revert it back